.github/workflows/build-image.yml

```
@@ -6,12 +6,15 @@ on:
 
 jobs:
   build-and-push-image:
-    runs-on: ubuntu-latest
+    runs-on: image-builder
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Build and push
       uses: byzanteam/jet-actions/fe-build-image@main
       with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+        registries: |-
+          ${{ secrets.ALIYUN_REGISTRY }},skylark,${{ secrets.ALIYUN_USERNAME }},${{ secrets.ALIYUN_KEY }}
```

`ALIYUN_SKYLARK_REGISTRY` 和 `ALIYUN_JET_REGISTRY` 直接可以取到这两个内容，不用手动 compose
```suggestion
          ${{ secrets.ALIYUN_SKYLARK_REGISTRY }}
```

---

OK